src/monero_ux_nano.c

```diff
@@ -79,21 +79,14 @@ void settings_show_25_words(void) {
     ui_menu_words_display(0);
 }
 /* -------------------------------- INFO UX --------------------------------- */
-unsigned int ui_menu_info_action(unsigned int value);
-
-UX_STEP_CB(ux_menu_info_1_step, nn, ui_menu_info_action(0),
+UX_STEP_CB(ux_menu_info_1_step, nn, (void)ui_menu_main_display,
```

Comment:
shouldn't it be this instead? (Not sure at all, but looking at was there before, and what's the implementation of `UX_STEP_CB`)
```suggestion
UX_STEP_CB(ux_menu_info_1_step, nn, ui_menu_main_display(),
```

Reply:
Good catch indeed :+1: